scripts/projects/TaikaiImporter.ts

```diff
@@ -0,0 +1,204 @@
+    let description = '';
+
+    // Get all text content for later use.
+    const allText = document.body.textContent || '';
+    
+    if (descriptionDiv) {
+      // Get all paragraphs from the description div.
+      const paragraphs = descriptionDiv.querySelectorAll('p');
+      let descriptionParts: string[] = [];
+      
+      for (const p of Array.from(paragraphs)) {
+        const text = p.textContent?.trim();
+        if (!text) continue;
+        
+        // Skip metadata paragraphs.
+        if (text.startsWith('Repository:') || 
+            text.startsWith('Video:') || 
+            text.startsWith('Contact:') ||
+            text.startsWith('Discord:') ||
+            text.startsWith('Telegram:')) {
+          continue;
+        }
+        
+        // Keep the original HTML content, just convert basic formatting.
+        let content = p.innerHTML
+          .replace(/<strong>(.*?)<\/strong>/gi, '**$1**')
+          .replace(/<em>(.*?)<\/em>/gi, '*$1*')
+          .replace(/<br\s*\/?>/gi, '\n');
+        
+        if (content) {
+          descriptionParts.push(content);
+        }
+      }
+      
+      description = descriptionParts.join('\n\n');
+    }
+    
+    // Fallback if we can't find the description div.
+    if (!description) {
+      const paragraphs = Array.from(document.querySelectorAll('p')).map(p => p.textContent?.trim()).filter(Boolean);
+      description = paragraphs.find(p => p && p.length > 50) || name;
+    }
+
+    // Add the hackathon project page link at the end.
+    description += `\n\nCheck out more on the hackathon's [project page](${projectUrl}).`;
+
+    // Extract video URL - be more specific about YouTube URL format.
+    const videoMatch = allText.match(/Video:\s*(https:\/\/youtu\.be\/[\w\-]+)/i) ||
+                      allText.match(/(https:\/\/youtu\.be\/[\w\-]+)/i);
+    let videoTutorial = '';
+    if (videoMatch) {
+      videoTutorial = videoMatch[1].replace(/[^a-zA-Z0-9\-\.:\/]/g, '');
+      videoTutorial = videoTutorial.split(/(?:\s|\(|Contact)/i)[0];
+    }
+    const tutorials = videoTutorial ? [{ 'Video demo': videoTutorial }] : [];
+
+    // Get demo URL - look for actual demo links, not just any link.
+    const demoUrlElement = Array.from(document.querySelectorAll('a')).find(
+      (el: HTMLAnchorElement): boolean => {
+        const text = el.textContent?.toLowerCase() || '';
+        const href = el.href?.toLowerCase() || '';
+        return (text.includes('demo') || text.includes('live') || text.includes('play')) && 
+               !href.includes('garden.taikai.network') && 
+               !href.includes('github.com') &&
+               !href.includes('youtu');
+      }
+    );
+    const demoUrl = demoUrlElement?.href || '';
+
+    // Get screenshots from both description and attachments.
+    const screenshots: string[] = [];
+
+    // Get images from description.
+    const descriptionImages = document.querySelectorAll('div.fr-view img');
```

Comment:
I tried adding https://taikai.network/cryptocanal/hackathons/ethdam2025/projects/cmaikn82p011a3pl1fpe6ntfm and the images/screenshots in carousel are not considered. And AFAIK they should not require the headless browser to fetch. Can you check it out?

Maybe just:

```suggestion
    const descriptionImages = document.querySelectorAll('img');
```